### ISSUE TITLE
Unreplaced `%s` format string in own profile view

### ISSUE THREAD

**Issue by xtaran** [engelsystem/engelsystem#1153]
On https://engelsystem.events.ccc.de/settings/profile is see the following string:

```
Das %s darf mir E-Mails senden (z.B. wenn sich meine Schichten ändern).
```

The `%s` is in there literally, but likely should have been replaced with a term like "Shift Operator" or "Engelsystem".

**Comment by MyIgel** [comment]
Hi, that problem is already fixed in 2e26f419 and will be deployed soon'isch :D

**Comment by MyIgel** [comment]
& deployed ^^
